crates/rust-analyzer/src/config.rs

```diff
@@ -56,6 +56,9 @@ mod patch_old_style;
 // parsing the old name.
 config_data! {
     struct ConfigData {
+        /// Whether to insert must_use derive macro while generating `as_` methods
+        /// for enum variants.
```

Comment:
```suggestion
        /// Whether to insert #[must_use] when generating `as_` methods
        /// for enum variants.
```
`#[must_use]` is not a derive macro, but an attribute

Reply:
Oops, nice catch